packages/devtools_app/lib/src/release_notes.dart

```diff
@@ -0,0 +1,222 @@
+        () {
+      setState(() {
+        markdownData = widget.releaseNotesController.releaseNotesMarkdown.value;
+      });
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Material(
+      child: Stack(
+        children: [
+          widget.child,
+          ReleaseNotes(
+            releaseNotesController: widget.releaseNotesController,
+            visibilityAnimation: visibilityAnimation,
+            markdownData: markdownData,
+          ),
+        ],
+      ),
+    );
+  }
+}
+
+class ReleaseNotes extends AnimatedWidget {
+  const ReleaseNotes({
+    Key key,
+    @required this.releaseNotesController,
+    @required Animation<double> visibilityAnimation,
+    @required this.markdownData,
+  }) : super(key: key, listenable: visibilityAnimation);
+
+  final ReleaseNotesController releaseNotesController;
+
+  final String markdownData;
+
+  @override
+  Widget build(BuildContext context) {
+    final animation = listenable as Animation<double>;
+    final theme = Theme.of(context);
+    return Positioned(
+      top: densePadding,
+      bottom: densePadding,
+      right: densePadding -
+          (_ReleaseNotesViewerState.viewerWidth - animation.value),
+      width: _ReleaseNotesViewerState.viewerWidth,
+      child: Card(
+        elevation: defaultElevation,
+        color: theme.scaffoldBackgroundColor,
+        clipBehavior: Clip.hardEdge,
+        shape: RoundedRectangleBorder(
+          borderRadius: BorderRadius.circular(defaultBorderRadius),
+          side: BorderSide(
+            color: theme.focusColor,
+          ),
+        ),
+        child: Column(
+          children: [
+            AreaPaneHeader(
+              title: const Text('What\'s new in DevTools?'),
+              needsTopBorder: false,
+              rightActions: [
+                IconButton(
+                  padding: const EdgeInsets.all(0.0),
+                  onPressed: () =>
+                      releaseNotesController.toggleReleaseNotesVisible(false),
+                  icon: const Icon(Icons.close),
+                ),
+              ],
+            ),
+            markdownData == null
+                ? const Text('Stay tuned for updates.')
+                : Expanded(
+                    child: Markdown(
+                      data: markdownData,
+                      onTapLink: (_, href, __) => launchUrl(href, context),
+                    ),
+                  ),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+class ReleaseNotesController {
+  ReleaseNotesController() {
+    _init();
+  }
+
+  ValueListenable<String> get releaseNotesMarkdown => _releaseNotesMarkdown;
+
+  final _releaseNotesMarkdown = ValueNotifier<String>(null);
+
+  ValueListenable<bool> get releaseNotesVisible => _releaseNotesVisible;
+
+  final _releaseNotesVisible = ValueNotifier<bool>(false);
+
+  void _init() {
+    if (server.isDevToolsServerAvailable && !isEmbedded()) {
```

Comment:
OOC, when is `isEmbedded` true?

Reply:
isEmbedded is true when DevTools is displayed inside the IDE tooling windows